src/Sulu/Bundle/AutomationBundle/Admin/AutomationContentNavigationProvider.php

```diff
@@ -62,9 +74,17 @@ public function getNavigationItems(array $options = [])
         $automation->setPosition($this->position);
         $automation->setAction('automation');
         $automation->setComponent('automation-tab@suluautomation');
-        $automation->setComponentOptions(['entityClass' => $this->entityClass]);
         $automation->setDisplay(['edit']);
 
+        $componentOptions = ['entityClass' => $this->entityClass];
+        if (array_key_exists('id', $options)) {
+            $locale = array_key_exists('locale', $options) ? $options['locale'] : null;
+            $futureTasks = $this->taskRepository->countFutureTasks($this->entityClass, $options['id'], $locale);
```

Comment:
The variable name sounds like an array of tasks. Would rename to something like `futureTaskCount`.